### ISSUE TITLE
Search by authors in gameslist.

### ISSUE THREAD

**Issue by popcan12** [sboxgame/issues#1690]
**What can't you do?**
Search by author for a gamemode.

**How would you like it to work?**
Possibly right under search all games, a field that says filter by author.

**What have you tried?**
In S&box dev if you know the author's handle you can go directly to their page but I feel not everyone who plays S&box will use the dev website.

**Additional context**

![image](https://user-images.githubusercontent.com/46109751/159063388-b13dc042-0238-458b-bff7-6003adc15852.png)
Basically like this, but maybe with the search all authors and search all games box closer together.

**Comment by handsomematt** [comment]
dupe issue
